<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseDeDonnee</class>
 <widget class="QWidget" name="BaseDeDonnee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BaseDeDonnee</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>535</width>
     <height>84</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Donnez un numéro de département</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditNumDepartements"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAvoirNom">
          <property name="text">
           <string>Avoir le nom du département</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelNomDepartement">
        <property name="text">
         <string>Nom du département</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="comboBoxRegions">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>Choisissez un région</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxDepartements">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>Choisissez un Département</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxVilles">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>Choisissez une Ville</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
